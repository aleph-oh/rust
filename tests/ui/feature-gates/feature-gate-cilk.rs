fn main() {
    cilk_spawn { 5 }; //~ ERROR cilk keywords are experimental [E0658]
    cilk_sync; //~ ERROR cilk keywords are experimental [E0658]
    cilk_scope { 5 }; //~ ERROR cilk keywords are experimental [E0658]
    cilk_for i in 0..10 {} //~ ERROR cilk keywords are experimental [E0658]
}
