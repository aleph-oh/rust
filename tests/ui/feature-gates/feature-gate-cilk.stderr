error[E0658]: cilk keywords are experimental
  --> $DIR/feature-gate-cilk.rs:2:5
   |
LL |     cilk_spawn { 5 };
   |     ^^^^^^^^^^
   |
   = help: add `#![feature(cilk)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error[E0658]: cilk keywords are experimental
  --> $DIR/feature-gate-cilk.rs:3:5
   |
LL |     cilk_sync;
   |     ^^^^^^^^^
   |
   = help: add `#![feature(cilk)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
